src/project_info.rs

```diff
@@ -266,6 +267,15 @@ fn is_application_prompt(default: Option<bool>) -> Result<bool> {
     Ok(value)
 }
 
+fn is_pixi_project_prompt(default: Option<bool>) -> Result<bool> {
+    let prompt_text =
+        "Pixi project\n  1 - Yes\n  2 - No\n  Choose from [1, 2]"
+            .to_string();
+    let value = boolean_prompt(prompt_text, default)?;
+
+    Ok(value)
+}
```

Comment:
This is currently defaulting the prompt to yes because the builder defaults `None` to 1.

```suggestion
fn is_pixi_project_prompt(default: Option<bool>) -> Result<bool> {
    let prompt_text = "Pixi project\n  1 - Yes\n  2 - No\n  Choose from [1, 2]".to_string();
    let default = if default.is_some() {
        default
    } else {
        Some(false)
    };
    let value = boolean_prompt(prompt_text, default)?;

    Ok(value)
}
```

Reply:
Sorry I missed this!

Reply:
No worries :smile: